Formula/rio.rb

```diff
@@ -0,0 +1,25 @@
+class Rio < Formula
+  desc "Hardware-accelerated GPU terminal emulator powered by WebGPU"
+  homepage "https://raphamorim.io/rio/"
+  url "https://github.com/raphamorim/rio/archive/refs/tags/v0.0.7.tar.gz"
+  sha256 "5a3fcdc6bbad3274286bc41c5225c9ae576ffcd13b80649247656c0f41422de9"
+  license "MIT"
+  head "https://github.com/raphamorim/rio.git", branch: "main"
+
+  depends_on "pkg-config" => :build
+  depends_on "rust" => :build
+  depends_on "fontconfig"
+  depends_on "freetype"
```

Comment:
There doesn't appear to be linkage with these dependencies. Can we try to ensure that they're used instead of `cargo` building a vendored version?

Reply:
good point. Hmm, Those dependencies are required for linux, I can try to build it without both deps in the CI.